interface/forms/vitals/C_FormVitals.class.php

```diff
@@ -398,12 +398,20 @@ public function default_action_process()
 
         $weight = $_POST["weight"];
         $height = $_POST["height"];
-        if ($weight > 0 && $height > 0) {
-            $_POST["BMI"] = ($weight / $height / $height) * 703;
+        // Validate input for weight and height
+        if (!is_numeric($weight) || $weight <= 0 || $weight > 1000 || !is_numeric($height) || $height <= 0 || $height > 120) {
+            $GLOBALS['error_message'] = "Invalid weight or height. Please check your input.";
+            $this->default_action(); // Render the form again with the error
+            return;
         }
 
+        // calculate BMI (weight in pounds, height in inches)
+        $_POST["BMI"] = round(($weight / $height / $height) * 703, 2);
+        // Assign BMI status
         // TODO: this should go into the vitals form...
-        if ($_POST["BMI"] > 42) {
+        if ($_POST["BMI"] > 50) {
```

Comment:
Is there a reason you changed this code logic?  Please refer to new updated guidelines of why you are adding in Severe Obesity and setting the level to be 50.